src/KDSoapServer/KDSoapServerObjectInterface.h

```diff
@@ -125,6 +127,17 @@ class KDSOAPSERVER_EXPORT KDSoapServerObjectInterface
      */
     virtual void processRequestWithPath(const KDSoapMessage &request, KDSoapMessage &response, const QByteArray &soapAction, const QString &path);
 
+    typedef QPair<QString/*name*/, QString/*value*/> HttpResponseHeaderItem;
```

Comment:
A struct (which means named members) is always more readable than a QPair, where one has to remember that .first means name and .second means value.